pkg/eestream/stripe.go

```diff
@@ -140,5 +150,21 @@ func (r *StripeReader) shouldWaitForMore(err error) bool {
 		return false
 	}
 	// check if there are more input buffers to wait for
-	return len(r.inmap)+len(r.errmap) < r.scheme.TotalCount()
+	return r.pendingReaders()
+}
+
+// combineErrs makes a useful error message from the errors in errmap.
+// combineErrs always returns an error.
+func (r *StripeReader) combineErrs() error {
+	if len(r.errmap) == 0 {
+		return Error.New("programmer error: no errors to combine")
+	}
+	errstrings := make([]string, 0, len(r.errmap))
```

Comment:
use `utils.CombineErrors` to preserve original errors

Reply:
I thought about using it, but it only prints the first 5 embedded errors. I think in this case it would be helpful to print all of them. Can you suggest how to do this with `utils.CombineErrors`?

Reply:
Hmm.. you know what, let's remove the limit, until there's a good reason to have the limit.